.github/workflows/ppr-ui-cd.yml

```diff
@@ -17,19 +17,14 @@ on:
         - test
         - sandbox
         - prod
-      tagname:
-        description: "Specify a previous version (git tag) to deploy"
-        required: false
-        default: ""
```

Comment:
Do we need to keep this in order to deploy protected branches?
I'm guessing this is just a label and not required?